cmd/tikv-ctl/src/util.rs

```diff
@@ -62,6 +62,19 @@ pub fn perror_and_exit<E: Error>(prefix: &str, e: E) -> ! {
     tikv_util::logger::exit_process_gracefully(-1);
 }
 
+// Check if the region is in the specified range
+pub fn included_region_in_range(region_range: (&[u8], &[u8]), check_range: (&[u8], &[u8])) -> bool {
```

Comment:
I suggest not using tuples to avoid later `xxx.0` and `xxx.1` which makes the code hard to read. Maybe it could look like this:

```rust
pub fn check_range_included(key_range: KeyRange, key_range_limit: KeyRange) -> bool {
```

`KeyRange` is an existing struct you can find here: https://github.com/tikv/tikv/blob/master/components/raftstore/src/store/util.rs#L392